starter-code/css/stylesheet.css

```diff
@@ -0,0 +1,104 @@
+.body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.navbar-brand {
+  font-weight: 800;
+  font-size: 36px;
+  text-transform: uppercase;
+}
+.container-fluid {
```

Comment:
Better practice is to create your own classes to manipulate, instead of directly changing the bootstrap ones. In this case, the change is minor, but in other cases it may cause some unexpected behavior. You could even create a reusable class like "white-bg-element"and use it declaratively for any element you want :)